Ladybird/AppKit/UI/TabController.mm

```diff
@@ -355,6 +463,24 @@ - (void)windowDidResize:(NSNotification*)notification
     }
 }
 
+- (BOOL)validateMenuItem:(NSMenuItem*)item
+{
+    if ([item action] == @selector(toggleLineBoxBorders:)) {
+        [item setState:(m_settings.should_show_line_box_borders) ? NSControlStateValueOn : NSControlStateValueOff];
```

Comment:
The parentheses around all of these booleans (such as `(m_settings.should_show_line_box_borders)`) are a bit superfluous